src/include/txn_inline.h

```diff
@@ -555,6 +561,8 @@ __wt_txn_visible_all(WT_SESSION_IMPL *session, uint64_t id, wt_timestamp_t times
 {
     wt_timestamp_t pinned_ts;
 
+    pinned_ts = 0U;
```

Comment:
We don't need the initialization as __wt_txn_pinned_timestamp should always populate pinned_ts.